docusaurus/docs/get-started/set-up-development-environment.mdx

```diff
@@ -439,3 +440,32 @@ Update the `scripts` in the `package.json` to use the extended Webpack configura
 -"dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
 +"dev": "webpack -w -c ./webpack.config.ts --env development",
 ```
+## Add a dependency for local development
+
+Grafana’s plugin development environment is designed to simplify the process of creating and testing plugins. Here’s how it operates:
+
+Plugin distribution path:
+
+The plugin’s dist folder (built plugin) is linked to `/var/lib/grafana/plugins/{{pluginId}}` in the Grafana server environment.
```

Comment:
I don't think is a good idea to be naming paths like this in the documentation because this can become very confusing quickly.

This text is true, but only if you are running grafana from a binary, in linux or mac. if you are running it with docker this is not longer valid because docker configuration often mount local folders, or if you clone and build the grafana repository it is also not the case. 

Better to link the official documentation https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#plugins

Reply:
done.